crates/sui-benchmark/tests/simtest.rs

```diff
@@ -68,7 +67,6 @@ mod test {
     }
 
     #[sim_test(config = "test_config()")]
-    #[ignore]
```

Comment:
the ignores that don't have the long comments should actually be left in place. these tests work but are not stable enough to run in CI.

Reply:
Is this still unstable after fixing the Narwhal issue?

Reply:
ok, i will put those back